config/application.rb

```diff
@@ -2,7 +2,7 @@
 
 require_relative 'boot'
 
-require 'rails'
+# require 'rails'
```

Comment:
But... this doesn't do anything? It just removes a commented out line that shows the whole rails suite isn't included, or am I missing something?

Reply:
yeah yeah it does not, I was planning to go forward with rails upgrade to 6.1 and exclude activestorage but then realized rspec-rails not compatible with 6.1. I will get rid of this commit, then do a rebase if we get 6.0.3.6 in